### DISCUSSION TITLE
Access signal cli via php?

### DISCUSSION THREAD

**Discussion by OSMmapper** [AsamK/signal-cli#1048] [Q&A]
I have installed the cli on my server and everything is running smoothly via console.
Is it possible to access the cli directly via php (maybe via shell_exec) and send/recieve messages?
So without API (eg: https://github.com/bbernhard/signal-cli-rest-api)?

**Comment by Tsjippy** [comment]
For me this one was a good start:
https://github.com/jigarakatidus/php-signal

**Comment by Tsjippy** [comment]
https://github.com/Tsjippy/php-signal/
